lib/suspenders/app_builder.rb

```diff
@@ -1,6 +1,14 @@
+require "forwardable"
+
 module Suspenders
   class AppBuilder < Rails::AppBuilder
     include Suspenders::Actions
+    extend Forwardable
+
+    def_delegators :heroku_adapter,
```

Comment:
I think we can also use the `delegate` helper that Rails provides here, for better readability.